### PR TITLE
release v0.3.0

### PR DESCRIPTION
**Pull request by sdemos**
release notes - 

**Warning**: updating between minor versions of this plugin will re-provision machines on existing managed cluster. See the [Ignition schema output section](https://github.com/coreos/terraform-provider-ct#ignition-schema-output) in the readme for more information

* Render Ignition Configs at Ignition 2.2.0
* Update `ct` to v0.8.0
* Add `snippets` field, a list of Container Linux Configs additively appended to the Container Linux Config in `content`

cc: @dgonyeo @dghubble

### CODE REVIEW

**Comment by sdemos** [comment]
github is showing those commits in the wrong order. `release v0.3.0` is last in git.

**Comment by dgonyeo** [comment]
You can get `2.2.0` Ignition configs with the ct `v0.8.0` release I made recently, not sure if it's worth bumping to that for this.

**Comment by dgonyeo** [comment]
Doing a `git rebase` with the `--ignore-date` flag should put them in the right order (github is trash at this)

**Comment by sdemos** [comment]
I think it's worth updating, since we have to release a new minor version every time we bump the ignition spec, so we might as well be on the latest when we do I think.

**Comment by sdemos** [comment]
have to update the tests to work with v2.2

File: CHANGES.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 Notable changes between releases.
 
+## v0.3.0 (2018-04-10)
+
+* Render Ignition Configs at Ingition 2.1.0
+* Update `ct` to v0.7.0
```
Comment (dgonyeo) [inline CHANGES.md]: These versions should be updated

Comment (sdemos) [reply CHANGES.md]: fixed

**Pull request merged**